modules/ossm-config-otel.adoc

```diff
@@ -99,41 +99,51 @@ spec:
 Once you verify that you can see traces, lower the `randomSamplingPercentage` value or set it to `default` to reduce the number of requests.
 ====
 
-. Deploy the `bookinfo` application in `bookinfo` namespace:
+. Create the `bookinfo` namespace by running the following command:
 +
 [source, terminal]
 ----
 $ oc create ns bookinfo
 ----
+
+. Depending on the update strategy you're using, enable sidecar injection in the namespace by running the appropriate command(s):
+
+.. If you are using the `InPlace` update strategy and your `Istio` resource is named `default`, then run the following command:
 +
 [source, terminal]
 ----
-$ oc label ns <namespace_name> istio.io/rev= # <1>
+$ oc label ns bookinfo istio-injection=enabled
 ----
 +
+.. If you are using the `RevisionBased` update strategy, then run the following commands:
++
+... Find your `<revision-name>` by running the following command:
++
 [source, terminal]
 ----
-$ oc apply -f https://raw.githubusercontent.com/istio/istio/release-1.23/samples/bookinfo/platform/kube/bookinfo.yaml -n bookinfo
+$ oc get istiorevisions.sailoperator.io
 ----
-<1> If you named your Istio resource `default` and are using the `InPlace` upgrade strategy, use `oc label ns bookinfo istio-injection=enabled`.
 +
-[NOTE]
-====
-To find your `<revision-name>`, run the following command:
-
+.Example output:
+[source,terminal]
+----
+NAME              TYPE    READY   STATUS    IN USE   VERSION   AGE
+default-v1-23-0   Local   True    Healthy   True    v1.23.0   3m33s
+----
++
+... Run the following command to label the namespace with the revision name to enable sidecar injection:
++
 [source, terminal]
 ----
-$ oc get istiorevisions.sailoperator.io
+$ oc label namespace bookinfo istio.io/rev=default-v1-23-0
```

Comment:
It seems that you are using an example value from the previous step's example output without making it explicit, and I think it would be helpful to point that out. I would suggest one of two ways of resolving this:

1. Putting a user-replaced value in this command, like <revision_name> after `istio.io/`, and then following it with another command (that has a ".Example command" title) using the actual example.
2. Adding a callout label here and adding in the description something like "specify your revision name, for example `default-v1-23-0`"